visualization/app/codeCharta/ui/dialog/dialog.globalSettings.component.ts

```diff
@@ -23,6 +23,7 @@ export class DialogGlobalSettingsController implements SettingsServiceSubscriber
 	}
 
 	public onSettingsChanged(settings: Settings, update: RecursivePartial<Settings>) {
+		//TODO: dead code. Subscription does not work with "controllerAs $ctrl"
```

Comment:
What's up with this TODO?

Reply:
Removed the TODO as the subscription works. Side note, which is also mentioned in the commit description:
- the constructor is called every time the dialog is opened
- and events only arrive when the dialog is opened